### PR TITLE
feat: create parser dell avamar

### PR DESCRIPTION
**Pull request by ikheifets-splunk**

### CODE REVIEW

**Review by RichardHEB — COMMENTED** [review]
Ilya, this is good to go, we tested it and it works great. Can you please merge to main?
Thanks,
Richard

File: docs/sources/vendor/Dell/avamar.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Dell Avamar
+
+## Key facts
+
+* Requires vendor product by source configuration
```
Comment (mstopa-splunk) [inline docs/sources/vendor/Dell/avamar.md:5]: @ikheifets-splunk can you change it to `MSG Format based filter`?

Comment (ikheifets-splunk) [reply docs/sources/vendor/Dell/avamar.md]: Sure, fixed. Very thanks I forgot to change it :)

**Review by mstopa-splunk — APPROVED** [review]

**Comment by RichardHEB** [comment]
Hi Ilya, any date on when this will merge with main? Thanks.

**Pull request merged**